src/go_emit.iced

```diff
@@ -746,18 +732,15 @@ exports.GoEmitter = class GoEmitter
     @untab()
     @output "}"
 
-  emit_preface : ({infile, types_only}) ->
-    @output "// Auto-generated #{if types_only then 'types' else 'types and interfaces'} using #{pkg.name} v#{pkg.version} (#{pkg.homepage})"
-    @output "//   Input file: #{path_lib.relative(process.cwd(), infile)}"
+  emit_preface : (infiles, json, {types_only} = {}) ->
+    @output "// Auto-generated to Go #{if types_only then 'types' else 'types and interfaces'} using #{pkg.name} v#{pkg.version} (#{pkg.homepage})"
+    if infiles.length == 1
+      @output "//   Input file: #{path_lib.relative(process.cwd(), infiles[0])}"
+    else
+      @output "//   Input files:"
+      for infile in infiles
+        @output "//   - #{path_lib.relative(process.cwd(), infiles)}"
```

Comment:
It'd be nice to alphabetize infiles here. So that two runs on the same set produce the same output even if the arg list is swapped around.

Reply:
I'm gonna do this is base_emitter, so that all emitters can take advantage of it.